doc/design/linc_permissions.md

```diff
@@ -0,0 +1,52 @@
+# Permissions for S3 Assets in LINC Archive
```

Comment:
```suggestion
# Access assets from private S3 bucket
```

Reply:
Updated the title for this development to focus on visualization and not compute.

Reply:
Updated via a commit!